org.uqbar.project.wollok/src/org/uqbar/project/wollok/model/WollokModelExtensions.xtend

```diff
@@ -435,4 +466,35 @@ class WollokModelExtensions {
 		val rootNodeName = e.name.trim
 		node.text.trim.equals(rootNodeName) && semanticsElements.contains(node.semanticElement.eClass.name) && doApplyRenameTo(e, node.semanticElement)
 	}
+	
+	def static dispatch expectsExpression(EObject e) { false }
```

Comment:
This method was intended only for parameters in constructors. So I will add a comment to avoid using for other purposes, and to enhance it...